src/Service/PartyHelper.php

```diff
@@ -294,6 +299,11 @@ public function getRelevantPartiesForHearingPostByCase(CaseEntity $case): array
             $counterparties[$relation->getParty()->getName().', '.$relation->getType()] = $relation->getParty();
         }
 
+        // Sort alphabetically
+        if (count($counterparties) > 1) {
+            uasort($counterparties, static fn ($a, $b) => strcmp($a->getName(), $b->getName()));
+        }
```

Comment:
Remove manual optimization and maybe use `<=>`.

Reply:
Removed check and used spaceship operator.

Reply:
Removed check and used spaceship operator.